src/core/expr/fexpr_minmax.cc

```diff
@@ -56,11 +56,9 @@ class FExpr_MinMax : public FExpr_ReduceUnary {
         case SType::INT16:
           return make<int16_t>(std::move(col), gby, is_grouped);
         case SType::INT32:
-        return make<int32_t>(std::move(col), gby, is_grouped);
         case SType::DATE32:
           return make<int32_t>(std::move(col), gby, is_grouped);
         case SType::INT64:
-        return make<int64_t>(std::move(col), gby, is_grouped);
```

Comment:
Not sure why this change was introduced, we don't need additional treatment of date32/time64 as internally there are the same as int32/int64.